object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/AutoValueSupport.java

```diff
@@ -106,7 +105,13 @@ public void handlePossibleBuilderBuildMethod(AnnotatedExecutableType t) {
           getAutoValueRequiredProperties(nextEnclosingElement, avBuilderSetterNames);
       AnnotationMirror newCalledMethodsAnno =
           createCalledMethodsForAutoValueProperties(requiredProperties, avBuilderSetterNames);
-      t.getReceiverType().addAnnotation(newCalledMethodsAnno);
+      // only add the new @CalledMethods annotation if there is not already a @CalledMethods
+      // annotation present
+      AnnotationMirror possibleBuildAnnotations =
+          t.getReceiverType()
+              .getAnnotationInHierarchy(
+                  atypeFactory.getQualifierHierarchy().getTopAnnotation(newCalledMethodsAnno));
+      if (possibleBuildAnnotations == null) t.getReceiverType().addAnnotation(newCalledMethodsAnno);
```

Comment:
It's considered bad style in Java code to use a one-line `if` statement, without braces. Please rewrite this to
```
      if (possibleBuildAnnotations == null) {
          t.getReceiverType().addAnnotation(newCalledMethodsAnno);
      }
```